tests/test_other.py

```diff
@@ -9086,7 +9086,7 @@ def test_supported_linker_flags(self):
     self.assertContained('warning: ignoring unsupported linker flag: `-rpath-link`', out)
 
     out = self.run_process([EMCC, path_from_root('tests', 'hello_world.cpp'),
-                            '-Wl,--no-check-features,-mllvm,-debug'], stderr=PIPE).stderr
```

Comment:
Probably just want to check multiple valid flags, can you just change `-debug` to any other llvm flag?